.travis.yml

```diff
@@ -8,8 +8,21 @@ php:
   - 5.6
   - hhvm
 
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.6
+      env: GUZZLE_VERSION=3.*
+    - php: 5.6
+      env: GUZZLE_VERSION=4.*
+    - php: 5.6
+      env: GUZZLE_VERSION=5.*
+    - php: 5.6
+      env: GUZZLE_VERSION=6.*
```

Comment:
this could be simplified:

- 5.3 builds are using Guzzle 3 already
- 5.4 build is using Guzzle 5 already
- 5.6 build is using Guzzle 6 already

Limiting the number of jobs in the matrix is a good idea as there is a quota of 5 concurrent jobs per github organization/user